Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -150,6 +150,7 @@ final class ProjectGenerator: ProjectGenerating {
                                     pbxproj: PBXProj) throws -> PBXProject {
         let defaultRegions = ["en", "Base"]
         let knownRegions = Set(defaultRegions + projectFileElements.knownRegions).sorted()
+        let attributes = project.organizationName.map { ["ORGANIZATION": $0] } ?? [:]
```

Comment:
👍

Reply:
aah sorry spotted one last issue :(

Thanks for adding the fixture! (manually) testing there I noticed the organization name was still not appearing in Xcode.

<img width="257" alt="Screen Shot 2020-03-13 at 8 17 54 AM" src="https://user-images.githubusercontent.com/11914919/76602630-42e78d80-6503-11ea-9897-2d9d6e8138ba.png">

I then manually typed in an organization name and looked at the diff:

```diff
 attributes = {
-	ORGANIZATION = Tuist;
+	ORGANIZATIONNAME = FooBar;
 };
```

The attribute needs to be `ORGANIZATIONNAME` by the looks of it.

```suggestion
        let attributes = project.organizationName.map { ["ORGANIZATIONNAME": $0] } ?? [:]
```

Reply:
doh! Thanks!